_sass/_layout.scss

```diff
@@ -295,6 +299,71 @@
         .hidden {
             display: none;
         }
+
+        .impact-totals {
+            .dashboard {
+                display: -webkit-box;
```

Comment:
do you want me to merge this and you set up a cleanup issue for across the site? 😏